server/src/lib/handlers/intermediate-file-validator.ts

```diff
@@ -1,31 +1,24 @@
 import { Request, Response } from "express";
-import path, { join } from 'path';
-import split2 from 'split2'
 import Busboy from 'busboy'
-import zlib from 'zlib'
-import * as validator from '../utils/validator';
-
-const typesPath = join(
-    path.dirname(require.resolve('@quintype/migration-helpers')),
-    'lib',
-    'editor-types.d.ts'
-  );
+import S3 from 'aws-sdk/clients/s3'
+import {validator, asyncReadStream, typesPath} from '../utils/validator'
+import { AWSCredentials } from '../../config'
 
 export function textInputValidator(req: Request, res: Response): void {
   const { type, data } = req.body;
-  let result: Object = {}
+  let result: any = {}
```

Comment:
Why is this `any` ?

Reply:
@Robert-Christopher changed the types. Please check